src/crypt_prov.rs

```diff
@@ -233,6 +234,43 @@ impl<'a> ImportOptions<'a> {
             res
         }
     }
+
+    /// Imports a PEM-encoded PKCS8 private key.
+    /// This functions decodes PEM blocks with or without "BEGIN PUBLIC KEY"
+    /// and "END PUBLIC KEY" headers
```

Comment:
should be "BEGIN/END PRIVATE KEY" right?

Reply:
yes it should